.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,64 @@
+name: "Build & Deploy DEV"
+
+on:
+  push:
+    branches:
+      - '**'
+      # - 'main'
+    paths:
+      - 'packages/**'
+      - '!packages/*/scripts/**'
+      - '!packages/models/**'
+      - '!packages/commons/**'
```

Comment:
questi non li escluderei, perché sono/saranno direttamente importati da catalog ed event, quindi vogliamo re-buildare

Reply:
Ok, avete già in mente come gestire le dipendenze comuni? 🤔  
Ad esempio si potrebbero pubblicare come package npm e selezionare le versioni nei `package.json` dei servizi che le utilizzano; ma è solo un'idea sul momento 😄

Reply:
Stiamo usando un monorepo, quindi tutti i servizi che ne hanno bisogno dipendono direttamente dai moduli comuni, senza bisogno di pubblicarli (a meno che non servano a a servizi esterni al monorepo, e.g. il frontend).

O forse non ho capito cosa intendi?

Reply:
Hai ragione, ora ho visto la configurazione dei workspaces 😄 
Modifico il match, grazie!